pom.xml

```
@@ -95,25 +95,26 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- Plugin versions. -->
-        <dokka-maven-plugin.version>1.8.10</dokka-maven-plugin.version>
+        <dokka-maven-plugin.version>1.9.10</dokka-maven-plugin.version>
         <maven-deploy-plugin.version>2.8.2</maven-deploy-plugin.version>
         <maven-gpg-plugin.version>3.0.1</maven-gpg-plugin.version>
-        <maven-project-info-reports-plugin.version>3.4.2</maven-project-info-reports-plugin.version>
+        <maven-project-info-reports-plugin.version>3.4.5</maven-project-info-reports-plugin.version>
         <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
-        <maven-surefire-plugin.version>3.1.2</maven-surefire-plugin.version>
+        <maven-surefire-plugin.version>3.2.1</maven-surefire-plugin.version>
         <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
 
         <!-- Library versions. -->
-        <equalsverifier.version>3.14</equalsverifier.version>
+        <equalsverifier.version>3.15.2</equalsverifier.version>
         <junit.version>4.13.2</junit.version>
-        <junit-jupiter.version>5.9.2</junit-jupiter.version>
-        <kotlin.version>1.8.10</kotlin.version>
-        <kotlin-test.version>1.8.10</kotlin-test.version>
-        <kotlin-test-junit.version>1.8.10</kotlin-test-junit.version>
+        <junit-jupiter.version>5.10.0</junit-jupiter.version>
+        <kotlin.version>1.9.20-RC2</kotlin.version>
```

This forces all clients to use this release candidate in production. Let's just use the latest release.

```suggestion
        <kotlin.version>1.9.10</kotlin.version>
```

same below